Main/game/game.js

```diff
@@ -170,11 +159,35 @@ export default class Game {
                 <button id="liar">Liar</button>
             `;
             showBidOptions.style.display = 'flex';
-            
         }, 3000)
+
+        this.showPlayerBidChoices();
         
     }
 
+    updateBidChoices(diceCount, diceValue) {
+        this.bidChoices = this.bidChoices.map((bidChoice) => {
```

Comment:
I think a `.forEach` here would be sufficient

Reply:
if switch to .forEach, code can't seem run because I require the return of each bidChoice to return in a new array. Can i stick to map?

Reply:
Don't understand why can't use .forEach.
Your reasoning don't make sense.

You are just manipulating the current array element `isSelected` no?

Then you can just do something like:
```js
this.bidChoices.map((bidChoice) => {
  bidChoice.isSelected = false;

  // if numOfDice is the same as diceCount, then look at dieFace value
  if (bidChoice.numOfDice < diceCount) {
      bidChoice.isSelected = true;
  }

  if (
      parseInt(bidChoice.numOfDice) === parseInt(diceCount) && 
      parseInt(bidChoice.dieFace) <= parseInt(diceValue)
  ) {
      bidChoice.isSelected = true;
  }
});
```

Also, do the `parseInt` outside fo the `.map`. By doing your way now, you are parsing the method argument as many times as there are numbers in the array. Why do X times when you can just do once?

Reply:
I am passing this function back into the invokeComputerTurn so I require a new array each time to update the choices available for both the computer as well as the player. Because after each bid ie dice count 4 x dice face 3, the bids "lower" should no longer be available. That's why need to use the .map function.

Will create a const for the pasteInt elements.